app/routes/auth/signin-oidc.js

```diff
@@ -46,10 +46,12 @@ module.exports = [{
       try {
         await auth.authenticate(request, session)
 
-        const personSummary = await getPersonSummary(request)
+        const apimAccessToken = await auth.getClientCredentials(request)
+
+        const personSummary = await getPersonSummary(request, apimAccessToken)
```

Comment:
something that was on my mind in general - can we add a configurable timeout to the API calls via Wreck please? I think we better do this before we forget - maybe 10seconds until we have a better value?

Reply:
Code updated - new config value added with default 10000 (10 secs)

`config.wreckHttp.timeoutMilliseconds`